app/javascript/modules/validate-forms.js

```diff
@@ -0,0 +1,21 @@
+// Example starter JavaScript for disabling form submissions if there are invalid fields
+(function () {
+  'use strict'
+  window.addEventListener('turbolinks:load', function() {
+    // Fetch all the forms we want to apply custom Bootstrap validation styles to
+    var forms = document.querySelectorAll('.needs-validation')
+
+    // Loop over them and prevent submission
```

Comment:
can you say more about using JavaScript for client-side validation over Motion?

Reply:
By default browsers do client side validation.  This removes the ugly browser default and adds the bootstrap way. We can also do validation with Motion.